generators/client/templates/angular/package.json.ejs

```diff
@@ -93,41 +94,32 @@ limitations under the License.
     "chai-string": "1.5.0",
     <%_ } _%>
     "codelyzer": "VERSION_MANAGED_BY_CLIENT_ANGULAR",
-    "copy-webpack-plugin": "VERSION_MANAGED_BY_CLIENT_ANGULAR",
```

Comment:
`jhipster-needle-add-assets-to-webpack` is not currently implemented any more, restore this plugin and add this needle to `webpack.custom.js`? Or add this needle to `angular.json`?

Reply:
Yes I'll check this, but I think the CLI also uses CopyWebpackPlugin so I don't know what the behaviour will be...

Reply:
@kaidohallik I managed to have this working by including the CopyWebpackPlugin in webpack.custom.js, I think it's easier to manage than having to manipulate the angular.json file, WDYT ?

Reply:
I'm having this error because the copy array is invalid..
```
An unhandled exception occurred: Invalid options object. Copy Plugin has been initialized using an options object that does not match the API schema.
 - options.patterns should be an non-empty array.
```

Maybe I can put a dummy folder inside ?

Reply:
Maybe this solution:
```
  const patterns = [
    // jhipster-needle-add-assets-to-webpack - JHipster will add/remove third-party resources in this array
  ];
  if (patterns.length > 0) {
    config.plugins.push(new CopyWebpackPlugin({ patterns }));
  }
```

Reply:
That was so obvious... Thanks alot ! 😄